### ISSUE TITLE
Connection but no UI

### ISSUE THREAD

**Issue by maacl** [witheve/Eve#186]
I am on OSX Yosemite. I see the following in the the terminal when running ```./run.sh```:

```
➜  Eve git:(master) ./run.sh
~/Dropbox/projects/tmp/Eve ~/Dropbox/projects/tmp/Eve
src/client.ts(129,95): error TS2339: Property 'message' does not exist on type 'CloseEvent'.
~/Dropbox/projects/tmp/Eve
~/Dropbox/projects/tmp/Eve ~/Dropbox/projects/tmp/Eve
multirust: using existing install for 'nightly-2015-08-10'
multirust: override toolchain for '/Users/mac/Dropbox/projects/tmp/Eve/runtime' set to 'nightly-2015-08-10'
     Running `target/release/server`
reading saved state took 0.0036471780040301383s
Connection from 127.0.0.1:50004
```

and the forllowing error in the Javascript browser console:

```
Failed to load resource: net::ERR_FILE_NOT_FOUND
client.ts:121 WebSocket connection to 'ws://localhost:2794/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED
client.ts:220 Uncaught ReferenceError: uiEditorRenderer is not defined
```

If I visit localhost:8080/editor I get the UI.

**Comment by dmorehouse** [comment]
I have this same issue as well.  After I browse to http://localhost:8080/editor and click New / Data my server crashes with the following output
Connection from 127.0.0.1:51801
thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Os { code: 22, message: "Invalid argument" } }', ../src/libcore/result.rs:732
stack backtrace:
   1:        0x103c6a935 - sys::backtrace::write::h6ddc342d792b0e00eys
   2:        0x103c6dfb0 - panicking::on_panic::h958ebf4d6905d53emex
   3:        0x103c5e332 - rt::unwind::begin_unwind_inner::hbd872a23a0e4a7d76Hw
   4:        0x103c5e98c - rt::unwind::begin_unwind_fmt::h6cb11d09fc9699d1cHw
   5:        0x103c6d97c - rust_begin_unwind
   6:        0x103c911d5 - panicking::panic_fmt::hbcb2030cdfd213bbTgC
   7:        0x103bc1de1 - server::send_event::ha9409291829be6cck4k
   8:        0x103bd3972 - server::handle_event::h0fa63162401f05e7Icl
   9:        0x103bd7348 - server::run::h6e76fd49812d796cKil
  10:        0x103aefc46 - main::haf3dba23e5e43d0aiaa
  11:        0x103c6d8dd - __rust_try
  12:        0x103c6ec4d - rt::lang_start::ha6eaa4ada71a9cdbh9w
An unknown error occurred

To learn more, run the command again with --verbose.

**Comment by dmorehouse** [comment]
This is fixed now.  It was fixed with commit - 6151e56014a9c45eae96b13d26ba45a9c2aeeef2

**Comment by maacl** [comment]
Confirm fixed.
